dspace/config/registries/openaire-cerif-types.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0"?>
+
+<dspace-dc-types>
+
+	<dspace-header>
+		<title>OpenAIRE Cris Types</title>
+	</dspace-header>
+
+	<dc-schema>
+		<name>oairecerif</name>
+		<namespace>https://www.openaire.eu/cerif-profile/1.1/</namespace>
```

Comment:
Why is this new schema required?  It seems like there's a lot of duplication with existing fields in DSpace.  For example:

* `oairecerif.author.affiliation` is similar to `person.affiliation.name`
* `oairecerif.funder` is similar to `project.funder.name`

Basically, it seems odd to me for us to keep adding more and more schemas that duplicate concepts.  It will also make it very complex to handle the same field in multiple schemas when we export from DSpace.  For instance, do we have to map both `oairecerif.funder` and `project.funder.name` to the same field in every case?

Is there a reason why we cannot use the Person/Publication/Project entity schemas here (and just make updates to those as needed)?